core/controllers/cron.py

```diff
@@ -185,3 +188,18 @@ def get(self):
                     jobs.MAPPER_PARAM_MAX_START_TIME_MSEC: max_start_time_msec
                 })
             logging.warning('Deletion jobs for auxiliary entities kicked off.')
+
+
+class CronAcceptStaleSuggestionsHandler(base.BaseHandler):
+    """Handler to accept suggestions that have no activity on them for
+    THRESHOLD_TIME_BEFORE_ACCEPT time.
+    """
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        """Handles get requests."""
+        suggestions = suggestion_services.get_all_stale_suggestions()
```

Comment:
Add a boolean flag in feconf (ENABLE_AUTO_ACCEPT_OF_SUGGESTIONS) that's set to False by default. We'll turn it on to True for the deployment. But I think we should have a mechanism to turn this off easily if we see issues in practice.

Reply:
Done.